spec/features/toggle_language_spec.rb

```diff
@@ -0,0 +1,216 @@
+describe 'ToggleLanguage', js: true do
+  let(:user) { create :user1 }
+
+  after(:each) do
+    page.reset!
+  end
+
+  feature 'Toggling the locale dropdown to change the language' do
+    describe 'When signed out' do
+      before(:each) do
+        visit root_path
+      end
+
+      it 'language can be toggled on the same page' do
+        within '#page_title' do
+          expect(page).to have_content 'if me is a community for mental health experiences'
+        end
+
+        select 'Español', from: 'locale'
+        within '#page_title' do
+          expect(page).to have_content 'if me es una comunidad para compartir experiencias de salud mental'
+        end
+
+        select 'English', from: 'locale'
+        within '#page_title' do
+          expect(page).to have_content 'if me is a community for mental health experiences'
+        end
+      end
+
+      it 'language can be toggled on different pages' do
+        select 'Español', from: 'locale'
+        within '#page_title' do
+          expect(page).to have_content 'if me es una comunidad para compartir experiencias de salud mental'
+        end
+
+        click_link('Acerca de')
+        within '#page_title' do
+          expect(page).to have_content 'Acerca de'
+        end
+
+        select 'English', from: 'locale'
+        within '#page_title' do
+          expect(page).to have_content 'About'
+        end
+      end
+    end
+
+    describe 'when signed out and then signed in' do
+      before(:each) do
+        visit root_path
+      end
+
+      after(:each) do
+        logout(:user)
+      end
+
+      it 'language can be toggled on different pages' do
```

Comment:
maybe this it block is more like "language persists when signing in"? i might even put that (and the one below) in its own `describe` that's "logging in" and "logging out"... that seems more descriptive/clear to me than "when signed out and then signed in"